query-service-impl/src/main/java/org/hypertrace/core/query/service/trino/converters/DefaultColumnRequestConverter.java

```diff
@@ -31,7 +31,7 @@
 
 /** Converts {@link QueryRequest} to Trino SQL query */
 class DefaultColumnRequestConverter implements ColumnRequestConverter {
-
+  private static final String MAP_KEY_OPERATOR = "element_at";
```

Comment:
nit: `element_at` is map function in trino world. Should we rename this to MAP_FUNCTION = ""element_at""

Reply:
done